recipes-kernel/linux/linux-yocto-onl-linux-5.15.y/bisdn-kmeta/cfg/bisdn-linux.scc

```diff
@@ -3,8 +3,11 @@
 patch 0001-net-rtnetlink-send-BONDING_INFO-events-down-to-users.patch
 patch 0001-net-ipv6-apply-IFLA_INET6_ADDR_GEN_MODE-immediately.patch
 
+# https://git.yoctoproject.org/yocto-kernel-cache/tree/
 include features/bpf/bpf.scc
 include features/cgroups/cgroups.scc
 include cfg/net/bridge.scc
 include cgl/cfg/net/ip_vs.scc
+
+# in this repo
```

Comment:
given that all the other references have the full path to the file, maybe this one should too